pandas/tests/tslibs/test_timestamps.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from pandas import Timestamp
+
+
+@pytest.mark.parametrize("kwargs", [{}, {"year": 2020}, {"year": 2020, "month": 1}])
+def test_timestamp_constructor_missing_keyword(kwargs):
```

Comment:
this can go in tests.scalar.timestamps.test_constructors

Reply:
moved, thanks for the heads up